packages-amd64-only.txt

```diff
@@ -0,0 +1,9 @@
+# These are packages that are only supported on amd64,
+# typically because they have not been updated since 2021.
+awless@cloudposse
+cfssl@cloudposse
+emailcli@cloudposse
+goofys@cloudposse
+rakkess@cloudposse
+tfenv@cloudposse
+tfmask@cloudposse
```

Comment:
I do believe we should deprecate 💯 of these from our base image. We can do that in a follow on PR. None of these are critical to have in the base image, and it's easy enough for users to mitigate it in their downstream images. Since 2.x has many breaking changes, this would be the time, no?

Reply:
@osterman The difficult one to remove is `goofys`, because we have commands `s3` and `s3fs` (that work together to mount an S3 bucket as if it were a local disk) that depend on it. That was my motivation to restore these commands and add stubs. Do we want to remove `s3` and `s3fs` as well?